SpaceShooter.py

```diff
@@ -57,6 +57,10 @@ def update(self):
             self.rect.x += self.speed
         if keys[K_a] and self.rect.x > 0:
             self.rect.x -= self.speed
+        if keys[K_w] and self.rect.y >=250:

```

Comment:
I want the player to be able to move up to 200 pixels
Change this line

Reply:
Ok

Reply:
done big boss